client/src/modules/MentorRegistry/components/MentorRegistryTableContainer.tsx

```diff
@@ -0,0 +1,349 @@
+        title: MentorsRegistryColumnName.Languages,
+        dataIndex: MentorsRegistryColumnKey.Languages,
+        render: tagsRenderer,
+        //TODO
+        filters: Object.values(UpdateUserDtoLanguagesEnum).map(language => {
+          return { text: language, value: language };
+        }),
+        width: 130,
+      },
+      {
+        key: MentorsRegistryColumnKey.StudentsLimit,
+        title: MentorsRegistryColumnName.StudentsLimit,
+        dataIndex: MentorsRegistryColumnKey.StudentsLimit,
+        width: 130,
+      },
+      {
+        key: MentorsRegistryColumnKey.PreferredLocation,
+        title: MentorsRegistryColumnName.PreferredLocation,
+        dataIndex: MentorsRegistryColumnKey.PreferredLocation,
+        sorter: stringSorter('githubId'),
+      },
+      {
+        key: MentorsRegistryColumnKey.Actions,
+        title: MentorsRegistryColumnName.Actions,
+        dataIndex: MentorsRegistryColumnKey.Actions,
+        render: (_: any, record: MentorRegistryDto) => (
+          <>
+            <Button type="link" size="small" onClick={() => handleModalDataChange('invite', record)}>
+              Invite
+            </Button>
+            {record.preselectedCourses.length && activeTab === MentorRegistryTabsMode.New ? (
+              <Button type="link" size="small" onClick={() => handleModalDataChange('re-send', record)}>
+                Re-send
+              </Button>
+            ) : null}
+            <Button type="link" size="small" onClick={() => handleModalDataChange('delete', record)}>
+              Delete
+            </Button>
+          </>
+        ),
+        width: activeTab === MentorRegistryTabsMode.New ? 210 : 140,
+      },
+    ];
+
+    return allColumns.filter(column => TABS[activeTab].find(tab => tab === column.dataIndex));
+  };
+
+  const handleTagClose = (removedTag: string) => {
+    const [removedTagName, removedTagValue] = removedTag.split(':');
+    switch (removedTagName) {
+      case MentorsRegistryColumnName.Tech:
+        {
+          setCombinedFilter(prevState => ({
+            ...prevState,
+            technicalMentoring: combinedFilter.technicalMentoring.filter(tag => tag !== removedTagValue.trim()),
+          }));
+        }
+        break;
+      case MentorsRegistryColumnName.PreferredCourses:
+        {
+          setCombinedFilter(prevState => ({
+            ...prevState,
+            preferredCourses: combinedFilter.preferredCourses.filter(
+              tag => courses.find(i => i.id === tag)?.name !== removedTagValue.trim(),
+            ),
+          }));
+        }
+        break;
+      case MentorsRegistryColumnName.Preselected:
+        {
+          setCombinedFilter(prevState => ({
+            ...prevState,
+            preselectedCourses: combinedFilter.preselectedCourses.filter(
+              tag => courses.find(i => i.id === tag)?.name !== removedTagValue.trim(),
+            ),
+          }));
+        }
+        break;
+      default:
+        break;
```

Comment:
I'd propose not to swallow this, maybe throw an error here

Reply:
fixed